lib/manageiq/schema/miq_command_recorder.rb

```diff
@@ -0,0 +1,21 @@
+module ManageIQ
+  module Schema
+    module MiqCommandRecorder
```

Comment:
Note:  Most of these are _**not**_ used, but I think this is intended for `def change` methods in a migration, so that they can be inverted if needed.  So there most likely is more to make this "feature complete" for future migrations, if any of these methods were to be used again, but arguably that is outside the scope of this PR.